tests/main/views/test_search.py

```diff
@@ -159,6 +164,38 @@ def test_should_render_summary_for_0_results_in_cloud_software_no_keywords(self)
         summary = find_search_summary(res.get_data(as_text=True))[0]
         assert '<span class="search-summary-count">0</span> results found in <em>Cloud software</em>' in summary
 
+    def test_should_render_suggestions_for_0_results(self):
+        self._search_api_client.search_services.return_value = {
+            "services": [],
+            "meta": {
+                "query": {},
+                "total": 0,
+                "took": 3
+            },
+            "links": {}
+        }
+
+        res = self.client.get('/g-cloud/search?lot=cloud-software')
+        assert res.status_code == 200
+        suggestion = find_0_results_suggestion(res.get_data(as_text=True))[0]
```

Comment:
hey wasn't too sure what you meant by this bit? could you expand by any chance?

If you meant that I should store the value of the response in a variable then use it on the next test then I intentionally did not do this. This is to make all tests independent and executable on their own.